stores/metadata/src/main/resources/db/migration/postgresql/V0_600_1__init.sql

```diff
@@ -7,6 +7,7 @@ create table transaction_metadata
     tx_hash               varchar(255) not null,
     label                 varchar(255),
     body                  text,
+    cbor                  text,
```

Comment:
For me this is not a problem but what would happen if a developer would already have a db, they would have to index from zero right?

Question is when is it sensible to start using actual migrations.

Reply:
0.0.11 has many breaking changes and need a full sync from beginning. In the pre-alpha phase (0.0.x), occasional full syncs might be necessary to minimize migration scripts. 

As we transition to beta (0.x.x) or at least one client app in production :), separate db migration scripts will be created for each release.